styles.css

```diff
@@ -178,6 +194,7 @@ li .nav-link {
   background-color: var(--font-color-1);
   height: auto;
   display: block;
+  background-image: url(./images/carbon.avif);
```

Comment:
Well done for adjusting the background image. However, it's yet to match the given design. Kindly make a copy of the image as closely as possible. You may use [this](https://user-images.githubusercontent.com/20554583/218251204-efa0057a-776f-4e31-93e0-890589a6f2ef.jpg).

Your design:
![image](https://user-images.githubusercontent.com/20554583/218251285-7ec544cb-c5ce-483a-b103-02876e4f657a.png)

Design guide:
![screenshot](https://user-images.githubusercontent.com/66230306/218248207-0af758cd-a37e-4453-ac17-2f7741776c8f.png)